tools/run_tests/xds_k8s_test_driver/framework/infrastructure/gcp/network_security.py

```diff
@@ -88,8 +88,9 @@ def from_response(cls, name: str,
                    rules=response.get('rules', []))
 
 
-class _NetworkSecurityBase(gcp.api.GcpStandardCloudApiResource,
-                           metaclass=abc.ABCMeta):
+class _NetworkSecurityBase(  # pylint: disable=abstract-method
```

Comment:
That's strange. This is an abstract class on it's own. Apparently pylint is bad at detecting it: https://stackoverflow.com/questions/39256350/pylint-cannot-handle-abstract-subclasses-of-abstract-base-classes

Let's put `pylint: disable` inside the class, and add the comment why this is disabled, with a TODO of removing it  once pylint is better at recognizing abstract classes inheriting from other abstract classes.

Reply:
Thank you!